ngraph/core/include/ngraph/op/normalize_l2.hpp

```diff
@@ -54,6 +54,5 @@ namespace ngraph
                 EpsMode m_eps_mode;
             };
         } // namespace v0
-        using v0::NormalizeL2;
```

Comment:
@lazarevevgeny is it ok from backward compatibility point of view? this `using` exists in previous releases. Or we just saying to use opset but not operations directly so we can remove it?

Reply:
@ilyachur , what do you think? My view on this is that we should keep it for backward compatibility.

Reply:
It is why I added `do not merge` label. And I will track this activity and merge only with OV2.0 changes.